generator/services.go

```diff
@@ -66,8 +66,18 @@ func (g *CaddyfileGenerator) getServiceTasksIps(service *swarm.Service, logger *
 		for _, task := range tasks {
 			if task.Status.State == swarm.TaskStateRunning {
 				hasRunningTasks = true
+				ingressNetworkFromLabel, overrideNetwork := task.Labels[IngressNetworkLabel]
```

Comment:
Might need to pull this label from `service` instead. IIRC, the task may have a different set of labels?

Reply:
Yup. Updated to use `service`